### PR TITLE
get country from service proposal

### PR DESCRIPTION
**Pull request by interro**

### CODE REVIEW

File: migrations/versions/56eff971505c_.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column('node', 'country')
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```
Comment (donce) [inline migrations/versions/56eff971505c_.py]: These comments for adjustment can be removed.

File: migrations/versions/56eff971505c_.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+"""empty message
```
Comment (donce) [inline migrations/versions/56eff971505c_.py]: Please add message here to make migrations more managable.
This could be as short as "remove country for node"

File: dashboard/model_layer.py [hunk 3]
```diff
@@ -9,6 +9,9 @@
 import dashboard.helpers as helpers
 
 
+sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
```
Comment (donce) [inline dashboard/model_layer.py]: Why is this needed?

File: models.py [hunk 4]
```diff
@@ -15,7 +15,6 @@ class Node(db.Model):
     __tablename__ = 'node'
     node_key = db.Column(db.String(IDENTITY_LENGTH_LIMIT), primary_key=True)
     ip = db.Column(db.String(45))
-    country = db.Column(db.String(255))
```
Comment (donce) [inline models.py]: This is still used in `nodes.html`

Comment (interro) [reply models.py]: fixed
